homeassistant/components/aftership/sensor.py

```diff
@@ -48,18 +48,13 @@
     {vol.Required(CONF_SLUG): cv.string, vol.Required(CONF_TRACKING_NUMBER): cv.string}
 )
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_API_KEY): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    }
-)
 
+async def async_setup_entry(hass, entry: ConfigEntry, async_add_entities) -> None:
+    """Set up the AfterShip Sensor from ConfigEntry."""
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
-    """Set up the AfterShip sensor platform."""
-    apikey = config[CONF_API_KEY]
-    name = config[CONF_NAME]
+    apikey = entry.data["api_key"]
+    name = entry.data["name"]
+    unique_id = entry.data["unique_id"]
```

Comment:
You can use `entry.entry_id` as unique id if there is no better option.

Reference: https://github.com/home-assistant/core/pull/35442#discussion_r422690314